### ISSUE TITLE
More gestures

### ISSUE THREAD

**Issue by GoogleCodeExporter** [anodynos/touchegg#16]
```
Hi,

I think it's great that you started this project. Maybe this will mean I can 
one day switch from OSX to Linux again. Have you ever looked at the OSX tool 
BetterTouchTool? It has loads of very handy gestures and I use it all the time. 
Some features it has that I think would be great for touchegg:

- Support for more complex gestures like "tip-tap" (e.g. put two fingers on the 
trackpad and tap with a third left or right of it)
- Possibility to configure different gestures for different applications
- Possibility to execute keyboard shortcuts or start programs with gestures
- Graphical user interface.

I know that you are just in the beginnings with touchegg but I really think you 
should take a look at BetterTouchTool.

Best regards,

Tobias

```

Original issue reported on code.google.com by `flugsche...@gmail.com` on 21 Feb 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
This was meant to be a feature suggestion, not a defect but I can't see how I 
can change this...
```

Original comment by `flugsche...@gmail.com` on 21 Feb 2011 at 6:52

**Comment by GoogleCodeExporter** [comment]
```
Hello!

Thanks for your sugestions:

> Support for more complex gestures like "tip-tap" (e.g. put two fingers on the 
trackpad and tap with a third left or right of it)

I'm using the uTouch library to develop Touchégg that not support more 
gestures. For the moment the Tap&Hold gesture (that is not a basic gesture 
provided by uTouch) is working, I will implement more complex-gestures in the 
future.

> Possibility to configure different gestures for different applications

Yes, is an excellent idea, I will add this feature in futures versions, thanks!

> Possibility to execute keyboard shortcuts or start programs with gestures

The "SEND_KEYS" action are now implemented, but I'm having problems to active 
Compiz/KWin, I don't know if this feature could be avaliable for next version.
The "RUN_COMMAND" is an action that I will add for next version.

> Graphical user interface.

Yes, I think that a GUI is now the most important feature for Touchégg, I'm 
working hard in this feature.


In conclusion, for the next version, Touchégg will have a GUI, a RUN_COMMAND 
action and, if I can make it work fine, a SEND_KEYS action.

In the future, I intend to incorporate more features, thanks for your ideas.

I haven't got a mac, but I will try to see (in screenshots) the BetterTouchTool 
options.

Thanks! More enhancements are wellcome!
```

Original comment by `jose.exposito89@gmail.com` on 21 Feb 2011 at 3:41
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

nice to hear that you are actively developing this and are open for ideas. 
Maybe I will have some time in the next year and will have a chance to look at 
the code and add some help. This depends on how my life will go in the next 
couple of months, though.

There's a documentation for BetterTouchTool at 
http://blog.boastr.net/?page_id=1619
This gives a pretty good overview of its features!

Also, the developer seems to be a very nice and helpful guy. Unfortunately he 
didn't open source his code (not as usual on OSX as it is on linux) and since 
it depends on the OSX libraries it might not help you so much, either. But he 
might be open for discussion if you would like somebody to talk to. 

Thanks so much for your contribution!

Tobias


```

Original comment by `flugsche...@gmail.com` on 21 Feb 2011 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
Thanks to you for your ideas and your help!
```

Original comment by `jose.exposito89@gmail.com` on 21 Feb 2011 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Hi!

Good news.

Now I added the two features you say me, "RUN_COMMAND" and "SEND_KEYS" and now, 
SEND_KEYS works fine with Compiz, Kwin or other programs.

Greetings!

PS - Set the issue a "done", because I'm working in the GUI (in the times when 
I don't add any feature that users asking me :P) and the others two features 
will delay to future versions ;)

```

Original comment by `jose.exposito89@gmail.com` on 23 Feb 2011 at 2:26
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
Nice, good work. I will try this. Looking forward to more gestures in the 
future!
```

Original comment by `flugsche...@gmail.com` on 23 Feb 2011 at 6:53

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hi! Now you can use in Natty (Ubuntu 11.04)
> - Possibility to configure different gestures for different applications
http://code.google.com/p/touchegg/issues/detail?id=22#c12

Yuo can download and install the source from svn, now of course the project 
have GUI, but I'm adding this features to it ;)

Greetings!

```

Original comment by `jose.exposito89@gmail.com` on 7 Apr 2011 at 12:21
